src/plugins/embeddable/public/lib/containers/embeddable_child_panel.tsx

```diff
@@ -41,19 +51,64 @@ export class EmbeddableChildPanel extends React.Component<EmbeddableChildPanelPr
   constructor(props: EmbeddableChildPanelProps) {
     super(props);
     this.state = {
-      loading: true,
+      firstTimeLoading: true,
     };
 
     this.mounted = false;
   }
 
+  private getEventStatus(output: EmbeddableOutput): EmbeddableRenderStatus {
+    if (output.rendered === true) {
+      return 'rendered';
+    } else if (output.loading === false) {
+      return 'loaded';
+    } else if (output.error !== undefined) {
+      return 'error';
+    } else {
+      return 'loading';
+    }
+  }
+
   public async componentDidMount() {
     this.mounted = true;
     const { container } = this.props;
 
+    let loadingTime = 0;
```

Comment:
As we use this variable to store the start time, should we call it `loadingStartTime` instead?